src/state_machine.zig

```diff
@@ -222,15 +223,15 @@ pub fn StateMachineType(comptime Storage: type, comptime constants_: struct {
 
         pub const Operation = enum(u8) {
             /// Operations reserved by VR protocol (for all state machines):
-            reserved,
-            root,
-            register,
+            reserved = 0,
+            root = 1,
+            register = 2,
 
             /// Operations exported by TigerBeetle:
-            create_accounts,
-            create_transfers,
-            lookup_accounts,
-            lookup_transfers,
+            create_accounts = constants_.vsr_operations_reserved + 0,
+            create_transfers = constants_.vsr_operations_reserved + 1,
+            lookup_accounts = constants_.vsr_operations_reserved + 2,
+            lookup_transfers = constants_.vsr_operations_reserved + 3,
```

Comment:
I kind of like avoiding auto-numbering for this since it helps ensure that the values are stable — reordering an enum is normally ok, but in this case it makes it clear that reordering is a breaking change.